apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -54,11 +56,53 @@ export function StudentAssessmentInstance({
           navPage: 'assessment_instance',
         })}
         <main id="content" class="container">
+          ${canDeleteAssessmentInstance
+            ? html`
+                <div class="alert alert-warning d-flex align-items-center">
+                  <span class="mr-3">
+                    Additional actions are available because you are viewing your own assessment
+                    instance and have student data editor permissions.
```

Comment:
```suggestion
                    instance and have student data view access.
```

Reply:
I made some changes in 408d7fe1c to make this text shorter and more direct.

Reply:
I think we're well into bikeshedding territory but "Instructor's own assessment instance special menu" just feels weird to me as a label.